recipe/meta.yaml

```diff
@@ -13,16 +13,19 @@ source:
     patches:
       - librt_linkage.patch  # [linux]
       - framework.patch  # [osx]
+      - macro_path_binary_relocation  # [linux]
 
 build:
-  number: 2
+  number: 3
   run_exports:
     - {{ pin_subpackage(name, max_pin='x') }}
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('c') }}
+    - git
+    - patch
```

Comment:
Btw, I had to add these two build dependencies otherwise building the package locally in my setup (without using docker) was failing. To be precise, my setup is already in a docker container, and running the docker script would require me to setup DIND.

Reply:
Good to know, but please remove them. The recipe should only contain the dependency required to build the recipe on the conda-forge CI, anything else is quite difficult to maintain.

Reply:
Done